service/models/User.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose'
+
+// Creating a User Schema
+const UserSchema = new mongoose.Schema({
```

Comment:
I like that you're thinking ahead about user details. Two questions:
1. Why use 'userName' instead of 'name'?
2. Will Users have one deck associated or more than one?